components/Applicant/Jobs/jobService.ts

```
@@ -118,40 +147,54 @@ export async function submitApplication(
         position: position,
+        cover_letter: coverLetter,
+        ideal_candidate: idealCandidate
       }),
     });
+
+    if (!applicationResponse.ok) {
+      const errorData = await applicationResponse.json();
+      console.error('Application submission failed:', errorData);
+      
+      // Handle specific error cases
+      if (errorData.error && errorData.error.includes("already applied")) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)